### ISSUE TITLE
Print an error message when trying to "start" a madwifi-ng VAP with airmon-ng

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#62]
**Reported by misterx on 23 Jan 2007 20:23 UTC**

```
linux-wueh:~ # airmon-ng


Interface       Chipset         Driver

wifi0           Atheros         madwifi-ng
eth1            Broadcom        bcm43xx
ath1            Atheros         madwifi-ng VAP (parent: wifi0)

linux-wueh:~ # airmon-ng  start ath1 6


Interface       Chipset         Driver

wifi0           Atheros         madwifi-ng
eth1            Broadcom        bcm43xx
ath1            Atheros         madwifi-ng VAP (parent: wifi0)

linux-wueh:~ # airmon-ng


Interface       Chipset         Driver

wifi0           Atheros         madwifi-ng
eth1            Broadcom        bcm43xx
ath1            Atheros         madwifi-ng VAP (parent: wifi0)

linux-wueh:~ # airmon-ng  start wifi0 6


Interface       Chipset         Driver

wifi0           Atheros         madwifi-ng
eth1            Broadcom        bcm43xx
ath1            Atheros         madwifi-ng VAP (parent: wifi0)
ath6            Atheros         madwifi-ng VAP (parent: wifi0)
```

**Comment by aircrack-ng** [comment]
**Comment by misterx on 23 Jan 2007 20:23 UTC**

Unfortunatly, airmon-ng cannot currently handle interface name change (that's why in the latest command, no interface is shown as being in monitor mode.

The check should maybe done with iwconfig for madwifi-ng.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 23 Jan 2007 20:23 UTC**

(In #166) Print an error message when trying to "start" a madwifi-ng VAP with airmon-ng (Closes: #62)

**Comment by aircrack-ng** [comment]
**Modified by misterx on 23 Jan 2007 20:23 UTC**
